puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/release/katello/3.13.groovy

```diff
@@ -0,0 +1,52 @@
+pipeline {
+    agent none
+
+    environment {
+        foreman_version = "1.22"
+    }
+
+    options {
+        timestamps()
+        timeout(time: 2, unit: 'HOURS')
```

Comment:
2 hours is not always enough for katello, i would raise it to 3 (not only here, but i won't ask for the rest in this pr)